### ISSUE TITLE
Logout confirm page is failing to log the user out on auth-server-wildfly

### ISSUE THREAD

**Issue by Aboullos** [keycloak/keycloak#11753]
### Describe the bug

Cookies tests are failing due to logout url confirm not working correctly

jenkins/job/universal-test-pipeline-server/1444/testReport/junit/org.keycloak.testsuite.broker/KcOidcBrokerEventTest/Run_tests___Cookies_Tests___cookies_tests___loginWithExistingUser/

### Version

18.0.0

### Expected behavior

User confirms logout, goes to user account and is redirected to the login page

### Actual behavior

User confirms logout, goes to user account and the user account is shown as if the user didn't log out

### How to Reproduce?

Cookies tests failing on auth-server-wildfly
- User is logged in
- User goes to logout url
- User click to confirm logout
- A screen is shown with a message that the logout was sucessful
- user goes to the account URL and the user information is shown as it didn't log out

**Comment by mposolda** [comment]
@miquelsi Yes, I will try to take a look. I afraid I have time for this after next week due other priorities and the planning F2F next week.
